setup.py

```diff
@@ -1,11 +1,13 @@
 try:
   from setuptools import setup, Extension
+  from setuptools.command.build_ext import build_ext
 except ImportError:
   from distutils.core import setup, Extension
+  from distutils.command.build_ext import build_ext
 import distutils.sysconfig
 from distutils.sysconfig import customize_compiler
 from distutils.command.build_clib import build_clib
-from distutils.command.build_ext import build_ext
+
```

Comment:
ah, that's interesting. Maybe we should detect this inside `hpy.devel` and raise an appropriate error instead of failing in an obscure way?

Reply:
ah ok I see that you have already started a discussion in https://github.com/hpyproject/hpy/issues/154